app/src/main/java/org/sopt/and/data/mapper/LoginMapper.kt

```diff
@@ -0,0 +1,9 @@
+package org.sopt.and.data.mapper
+
+import org.sopt.and.data.remote.model.response.BaseResponse
+import org.sopt.and.data.remote.model.response.ResponseLoginDto
+import org.sopt.and.domain.model.Token
+
+fun BaseResponse<ResponseLoginDto>.toDomain(): Token {
+    return Token(token = this.result.token)
+}
```

Comment:
이렇게도 가능합니다!(이미 아시겠지만 그래도 쇽 남겨보고싶었어요..ㅎ)
```suggestion
fun BaseResponse<ResponseLoginDto>.toDomain(): Token =
    Token(token = this.result.token)
```